### PR TITLE
Handle crash sendchat

### PR DESCRIPTION
**Pull request by wonderingabout**
moving this out of #145 

so this PR is on top of #145 

see #145 for talks details

@roy7 @Dorus @windo 
tested to work, but currently the log and senchats are not what we want (wrong order of messages + two resigning messages when we resign only once)

### CODE REVIEW

**Comment by roy7** [comment]
#145 is merged, can you do a git pull?

**Comment by wonderingabout** [comment]
yes ok @roy7 

but here is a note for @windo to remember to look why log acts in a weird way :+1:

**Comment by wonderingabout** [comment]
done @roy7 :+1:

File: game.js [hunk 1]
```diff
@@ -299,7 +300,8 @@ class Game {
         }
 
         if (!this.ensureBotStarted(botError)) {
-            this.log("Failed to start the bot, can not make a move");
+            this.log("Failed to start the bot, can not make a move, trying to restart");
+            this.sendChat("Failed to start the bot, can not make a move, trying to restart"); // we notify user of this in ingame chat
```
Comment (windo) [inline game.js]: I don't think the user needs to know about this. But don't particularly mind.

Comment (wonderingabout) [reply game.js]: @windo 
imagine if our bot is slow to start/restart

then the user may have to wait many minutes without knowing what is going on

i think in that case, waiting say 30 seconds instead with a message after 30 seconds is much better :+1:

**Pull request merged**